FlavorTrack/Core/BusinessInfoVC/BusinessInfoVC.swift

```diff
@@ -31,85 +31,89 @@ final class BusinessInfoVC: UIViewController, LoadableScreen {
 		self.detailView = .init()
 		self.mapHostingView = .init()
 	}
-	
+
 	required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
-	
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
 		view.backgroundColor = .systemBackground
 		title = business.name
-		
+
 		configNavigationBarButtons()
 		layoutUIElements()
 		configUIElements(for: business)
     }
 }
 
 private extension BusinessInfoVC {
-	
+
 	private func configNavigationBarButtons() {
 		let doneBarBtn: UIBarButtonItem = .init(barButtonSystemItem: .done, target: self, action: #selector(dismissVC))
 		doneBarBtn.isAccessibilityElement = true
-		doneBarBtn.accessibilityLabel = NSLocalizedString("Done and dismiss view", comment: "The accessibility label of the Done navigation bar button")
+		doneBarBtn.accessibilityLabel = NSLocalizedString(
+			"Done and dismiss view",
+			comment: "The accessibility label of the Done navigation bar button")
 		navigationItem.setRightBarButtonItems([doneBarBtn], animated: true)
-		
+
 		let favBarBtn: UIBarButtonItem = .init(image: SFSymbols.star, style: .plain,
 											   target: self, action: #selector(favoriteButtonClicked))
 		favBarBtn.isAccessibilityElement = true
-		favBarBtn.accessibilityLabel = NSLocalizedString("Add to favorites", comment: "The accessibility label of the Favorite navigation bar button")
+		favBarBtn.accessibilityLabel = NSLocalizedString(
+			"Add to favorites",
+			comment: "The accessibility label of the Favorite navigation bar button")
```

Comment:
Try placing ending parentheses in the next line:

`favBarBtn.accessibilityLabel = NSLocalizedString(
			"Add to favorites",
			comment: "The accessibility label of the Favorite navigation bar button"
)`